deploy/roles/psql-authnz/tasks/main.yml

```diff
@@ -5,18 +5,7 @@
 
 - name: Install dependencies
   yum:
-    name: "{{item}}"
+    name: ['python-devel', 'libevent-devel', 'gcc', 'gcc-c++', 'kernel-devel', 'libxslt-devel', 'libffi-devel', 'openssl-devel', 'openldap-devel']
```

Comment:
For these long lists an item per line would be more readable

Reply:
I've now got the entries on separate lines when there are 3 or more of them.